common/static/common/js/components/views/paginated_view.js

```diff
@@ -26,7 +26,7 @@
     ], function (Backbone, _, PagingHeader, PagingFooter, ListView, paginatedViewTemplate) {
         var PaginatedView = Backbone.View.extend({
             initialize: function () {
-                var ItemListView = ListView.extend({
+                var ItemListView = this.listViewClass.extend({
```

Comment:
Nit: Add a comment explaining why we're referencing this.listViewClass instead of ListView